packages/tools/package/test/module/changelog/changelog.fixtures.js

```diff
@@ -0,0 +1,39 @@
+const packagesData = {
+  webex: {
+    data: {
+      location: '/Users/shreysh2/Desktop/Repo/webex-js-sdk/packages/webex',
```

Comment:
Why is your system details here? Should we consider replacing it with a generic one?

Reply:
Removed.